Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -372,6 +382,10 @@ void InterfacePane::AddDescriptions()
       QT_TR_NOOP("Shows on-screen display messages over the render window. These messages "
                  "disappear after several seconds."
                  "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static const char TR_OSD_FONT_SIZE_DESCRIPTION[] = QT_TR_NOOP(
+      "Changes the font size of the On Screen Display. Affects features such as performance "
```

Comment:
The fact it mentions performance statistics is strange to me.  I guess you mean FPS/VPS?  Because the graph is hardcoded.